### PR TITLE
Fix downloaded files moved to citation key dir

### LINKED ISSUE #6589: Download of files put them in subfolders
Just a quick note for now: after downloading a linked online file, it was put in the folder `standardRoot/citationkey/`. Not sure why and under which conditions this is reproducible.

### PR DESCRIPTION
**Pull request by Siedlerchr**
Fixes #6589 

- [x] Change in CHANGELOG.md described (if applicable)
- [x] Tests created for changes (if applicable)
- [x] Manually tested changed features in running JabRef (always required)
- [ ] Screenshots added in PR description (for UI changes)
- [ ] [Checked documentation](https://docs.jabref.org/): Is the information available and up to date? If not created an issue at <https://github.com/JabRef/user-documentation/issues> or, even better, submitted a pull request to the documentation repository.

### CODE REVIEW

File: src/test/java/org/jabref/logic/cleanup/MoveFilesCleanupTest.java [hunk 1]
```diff
@@ -111,6 +111,19 @@ void movesFileWithFileDirPattern() throws Exception {
         assertTrue(Files.exists(fileAfter));
     }
 
+    @Test
+    void doesNotmMovesFileWithEmptyFileDirPattern() throws Exception {
```
Comment (systemoperator) [inline src/test/java/org/jabref/logic/cleanup/MoveFilesCleanupTest.java:115]: Are there some typos? (`doesNotMoveFile`)

Comment (Siedlerchr) [reply src/test/java/org/jabref/logic/cleanup/MoveFilesCleanupTest.java]: Thanks, indeed.

File: src/test/java/org/jabref/logic/util/io/FileUtilTest.java [hunk 2]
```diff
@@ -357,15 +357,26 @@ void invalidFilenameWithoutExtension() {
...
         entry.setField(StandardField.AUTHOR, "A. Åuthör and Author, Bete");
 
-        assertEquals("PDF/1998/Åuthör/1234 - mytitle",
-                FileUtil.createDirNameFromPattern(null, entry, fileNamePattern));
+        assertEquals("PDF/1998/Åuthör/1234 - mytitle", FileUtil.createDirNameFromPattern(null, entry, fileDirPattern));
+    }
+
+    @Test
+    void testGetLinkedDirNamePatternEmpty() {
+        // bibkey - title
```
Comment (koppor) [inline src/test/java/org/jabref/logic/util/io/FileUtilTest.java:372]: Can this line be removed? Seems to be a copy and paste issue.

File: src/main/java/org/jabref/logic/util/io/FileUtil.java [hunk 3]
```diff
@@ -284,7 +284,7 @@ public static String createDirNameFromPattern(BibDatabase database, BibEntry ent
         String targetName = BracketedPattern.expandBrackets(directoryNamePattern, ';', entry, database);
 
         if (targetName.isEmpty()) {
-            targetName = entry.getCiteKeyOptional().orElse("default");
+            return targetName;
```
Comment (koppor) [inline src/main/java/org/jabref/logic/util/io/FileUtil.java:287]: Wow, what a mistake in the code. I checked the method names and comment. The change seems to be valid.

**Pull request merged**